libraries/AP_EFI/AP_EFI_MAV.h

```diff
@@ -0,0 +1,30 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <GCS_MAVLink/GCS_MAVLink.h>
+
+#include "AP_EFI.h"
+#include "AP_EFI_Backend.h"
+
+class AP_EFI_MAV : public AP_EFI_Backend {
+public:
+    AP_EFI_MAV(AP_EFI &_frontend);
+
+    void update() override;
+
+    void handle_EFI_message(const mavlink_message_t &msg) override;
```

Comment:
Rather than reaching directly to the backend is better to have wrapper function in the base class. This can check that the backend exists, and in the future can cope with multiple backends. Much like the update method.